addons/pkg-npm/jaxrs/src/main/java/org/commonjava/indy/pkg/npm/jaxrs/NPMContentAccessHandler.java

```diff
@@ -396,8 +397,12 @@ private void generateHttpMetadataHeaders( final Transfer transfer, final List<Tr
             return;
         }
 
-        Response responseWithLastModified =
-                Response.fromResponse( response ).lastModified( new Date( transfer.lastModified() ) ).build();
+        Response responseWithLastModified = Response.fromResponse( response )
+                                                    .header( "TRANSFER-ENCODING", null )
```

Comment:
Can we change these and define them as new enum entries in ApplicationHeader (in indy-api) then use them from there? Not critical, and I'll merge this without it, but it's a cleanup step we should do.